src/lib/Microsoft.Fx.Portability.Reports.Excel/ExcelReportWriter.cs

```diff
@@ -26,7 +26,7 @@ public ExcelReportWriter(ITargetMapper targetMapper)
 
         public ResultFormatInformation Format { get { return _formatInformation; } }
 
-        public void WriteStream(Stream stream, AnalyzeResponse response)
+        public void WriteStream(Stream stream, AnalyzeResponse response, AnalyzeRequest request)
```

Comment:
What information do you need from the AnalyzeRequest? This may make it difficult to incorporate into the service

Reply:
I use that information to create the assembly reference graph. When we first analyze the assemblies (before we send them back) we capture enough information about their dependencies to allow us to generate the DGML graph. 

That information can be computed before we send the request as well -- and we can probably stash it somewhere in order for the information to survive the round-trip.

Reply:
Keep in mind that the writers are inserted as output formatters in the MVC pipeline, so it just gets the output. If you can find a way to have it go through the AnalyzeResponse, that would be ideal, as I'm not sure how we could easily get the request without some jumping through hoops (maybe accessing the HttpContext or something, but I'm not as familiar with ASP.NET as ASP.NET Core). You can annotate it with [JsonIgnore] so it won't be serialized down if application/json is requested